planet/api/models.py

```diff
@@ -180,105 +134,277 @@ def name(self):
         :returns: name of this resource
         :rtype: str
         '''
-        return utils.get_filename(self.response)
+        name = (_get_filename_from_headers(self.response.headers) or
```

Comment:
These `*filename*` functions don't look like they belong in this module, and I'm curious why they seem to have been moved here?

Reply:
I moved it because its only used in this module and I dislike using `utils` as a dumpster for random functionality.

Reply:
In that case, how about moving them to somewhere more relevant to their functionality? To me, `planet.api.models._get_filename_from_headers` is more astonishing/surprising than `planet.api.utils._get_filename_from_headers`, for example.

Reply:
My feedback might be to raise these methods up a level, but since they're internal-use-only it doesn't really matter. I am strong +1 on _not_ putting them back in a catch-all utils. This is fine as-is.